### DISCUSSION TITLE
Survey Likert plugin autocomplete parameter

### DISCUSSION THREAD

**Discussion by caregrad** [jspsych/jsPsych#2952] [Q&A] [answered]
Dear all, 

Could you help me with the autocomplete parameter for the survey-Likert plugin?

I am doing an experiment with bilinguals where I ask them to evaluate the various parameters of the word using the Likert scale. 
Also, I want them to have the possibility to choose 'I don't know the word' on the scale.  My idea was to set up ' 'I don't know the word'  as a default parameter, which is already autocompleted: to prevent them from doing it several times. 

Is it possible to do it using the autocomplete parameter? And if yes, how? I searched, but I couldn't find the answer. 

Thank you!

I want the slide looks like this:

![2023-02-03_15-53-50](https://user-images.githubusercontent.com/79867613/216555871-59f3fda9-cf81-4475-b83b-3bbd0e364af8.png)

**Comment by ChristopheBossens** [comment, marked as answer]
I think the autocomplete parameter has a different purpose than what you need (see this link: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete). The behavior it enables is what you typically see on address forms where you fill out your name and it will automatically complete your address as well.

To achieve what you need, you can adjust the code of the plugin if you are comfortable with that (if not let me know and I can send an adjusted plugin file). If you go to the section where the options string for the radio input field is defined, add the following code after line 150 (I am assuming jsPsych 7.2 btw):

![image](https://user-images.githubusercontent.com/7238485/218443150-31f658da-1a44-4353-b8b0-273c0ef8c8b3.png)

This will add the checked attribute to the first radio button that is generated.

From an experimental point of view, don't you think it is better to have the participant select this option explicitly for each word? Else they might simply click the further button to complete the page since all the options have already been set for them?

**Reply by caregrad** [reply]
Hey! Thank you very much for your answer! I will definitely need the solution in my experiments.

As for the last point, yes, we also thought about it. In the end, we came to the decision to get rid of this on the Likert scale (because the problem you mentioned) and make two buttons below the Likert scale: “I don’t know the word” and “Submit answers”.
